src/mame/layout/timetrv.lay

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0"?>
+<!--
+license:BSD-3-Clause
```

Comment:
Is there a specific reason for a BSD-licensed layout?  We generally use CC0 for internal layouts to reduce the hassle for people using them as examples to follow.  If there’s a reason for it, it’s fine though.

Reply:
I did notice that all of the internal layouts I had looked at were CC0, but I don't recall seeing the justification for it stated until now. Completely understandable, and I've adjusted the license to be CC0.